src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -296,6 +296,13 @@ private CheckResult parseGatorGraderCommand(GatorGraderCommand cmd, boolean incl
             false,
             "The " + unrec + " check is not supported"
         );
+      } else if (output.contains("--reach")) {
+        result = new CheckResult(
+            cmd,
+            "Contains --reach",
+            true,
+            "this might work"
```

Comment:
This runs into the same issue as in `CheckResult`, where the results for reach goals are stored with incorrect outcomes. What really needs to happen is that if a check is a reach goal, it doesn't count towards failures.